src/github/handlers/issue-comment-created.ts

```diff
@@ -1,9 +1,212 @@
+import { Manifest } from "@ubiquity-os/plugin-sdk/manifest";
 import { GitHubContext } from "../github-context";
+import { PluginInput } from "../types/plugin";
+import { isGithubPlugin, PluginConfiguration } from "../types/plugin-configuration";
+import { getConfig } from "../utils/config";
+import { getManifest } from "../utils/plugins";
+import { dispatchWorker, dispatchWorkflow, getDefaultBranch } from "../utils/workflow-dispatch";
 import { postHelpCommand } from "./help-command";
 
 export default async function issueCommentCreated(context: GitHubContext<"issue_comment.created">) {
-  const body = context.payload.comment.body.trim();
-  if (/^\/help$/.test(body)) {
+  const body = context.payload.comment.body.trim().toLowerCase();
+  if (body.startsWith(`@ubiquityos`)) {
```

Comment:
I wonder if we should offer a short hand syntax. 

As I understand @UbiquityOS wouldn't automatically populate on the GitHub UI. 

Mixed thoughts on this idea, but it might be more ergonomic to use with something like @u or @os 

Also this next idea could be out of scope but wondering if we should intercept all comments and run commands on behalf. Would be perfect if we could handle those assigns for the newcomers asking to work on tasks for example.

Reply:
> I wonder if we should offer a short hand syntax.
> 
> As I understand @UbiquityOS wouldn't automatically populate on the GitHub UI.
> 
> Mixed thoughts on this idea, but it might be more ergonomic to use with something like @U or @os

Yes, it's a bit cumbersome to always type out @UbiquityOS so a short hand tag would be great but we will be tagging another person :D

Reply:
It might be interesting to have some type of local logic (perhaps non LLM) to determine if the comment is likely asking for some type of action. Accuracy can be pretty low and in theory it could still cut quite a bit of costs. 

Also with mini models costs might already be cheap enough for it to be feasible. We would need to run projections I suppose.

Reply:
A solution would be to hook into the issue author's name. At least under the issue this would be intuitive. Under the pull I'm not sure if it makes sense to hook into the pull author, as they are generally not the person you would ask these sort of questions to.